sources/packages/api/src-sql/Institution/Create-Institution.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE IF NOT EXISTS institutions(
+  id SERIAL PRIMARY KEY, 
+  legal_operating_name VARCHAR(64) NOT NULL, 
+  operating_name VARCHAR(64) NULL,  
+  primary_phone VARCHAR(64) NOT NULL,  
+  primary_email VARCHAR(64) NOT NULL,  
+  institution_type VARCHAR(64) NOT NULL,  
+  website VARCHAR(64) NOT NULL,  
+  regulating_body VARCHAR(64) NOT NULL,  
```

Comment:
Are we gonna be creating an enum for this or it will accept any string received from the UI?

Reply:
My thought is that Enum should be at the front end. It will come to API as a string.

Reply:
I believe that the API should know the values. How are we going to enforce that the values received by the API are valid?

Reply:
For Temp Solution, it was agreed between the team that having an enum on the front end and API should have a string. There is the possibility of wrong values being sent into the API and wrong things will be stored. This will be revisited later.